cnew.py

```diff
@@ -143,21 +170,35 @@ def mainfunc(username, password, chatroom, servaddr):
             else:
                 print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <red>An invalid chatroom password was entered</red>"))
                 sys.exit()
-        print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Chatroom identity : " + chatroom + "</seagreen>"))
-        print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Chatroom password : " + password + "</seagreen>"))
-        print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Share the chatroom identity and password to add members!</seagreen>"))
-        print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Your conversations are protected with end-to-end encryption</seagreen>"))
+        print_formatted_text(
+            HTML("[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Chatroom identity : " + chatroom + "</seagreen>"))
+        print_formatted_text(
+            HTML("[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Chatroom password : " + password + "</seagreen>"))
+        print_formatted_text(HTML(
+            "[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Share the chatroom identity and password to add members!</seagreen>"))
+        print_formatted_text(HTML(
+            "[" + obtntime() + "] " + "SNCTRYZERO > <seagreen>Your conversations are protected with end-to-end encryption</seagreen>"))
         asyncio.get_event_loop().run_until_complete(hello(servaddr, username, chatroom, password))
     except KeyboardInterrupt as EXPT:
         print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <red>Leaving SNCTRYZERO...</red>"))
         sys.exit()
     except OSError as EXPT:
-        print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <red>A connection to the server could not be established</red>"))
+        print_formatted_text(HTML(
+            "[" + obtntime() + "] " + "SNCTRYZERO > <red>A connection to the server could not be established</red>"))
         sys.exit()
     except websockets.exceptions.ConnectionClosedError as EXPT:
-        print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <red>A connection to the server was lost</red>"))
+        print_formatted_text(
+            HTML("[" + obtntime() + "] " + "SNCTRYZERO > <red>A connection to the server was lost</red>"))
         sys.exit()
 
 
+def sig_handler(signum, frame):
+    print_formatted_text(
+        HTML("[" + obtntime() + "] " + "SNCTRYZERO > <red>SIGABRT{} is being called</red>".format(signum)))
+    raise KeyboardInterrupt
+
+
+signal.signal(signal.SIGABRT, sig_handler)
```

Comment:
I don't think we would be needing this anymore. Are there ways to trigger a SIGABRT from inside the application?

Reply:
yes, if client app is being killed, it's still sending Exited information to server. 
<img width="1435" alt="Screen Shot 2020-10-17 at 13 45 38" src="https://user-images.githubusercontent.com/11618498/96330411-1ab6a300-107f-11eb-8c2c-d8a5b313ddec.png">